### DISCUSSION TITLE
How to get a db_name for tsh db login

### DISCUSSION THREAD

**Discussion by tempus-krupa-patel** [gravitational/teleport#46172] [Q&A]
I'm trying to use `tsh db login`, which requires a `db_name`. When I run `tsh db ls --format json`, I can see the instance name and other details. In some cases, I can use the `system name` as the `db_name`, but this approach doesn't work when there are multiple databases under a single GCP project. How can I determine the correct `db_name` for a `tsh db login` in such scenarios?

**Comment by webvictim** [comment]
The `db_name` is not a Teleport concept - it's the name of the database in the system you're logging into. In other words, it's something that would be dictated and set by the business use case/DBAs on your side.

For example, with MySQL, if you run `mysql -D my-database -u root -p`, the `db_name` there is `my-database`.

What database are you using, out of interest?

**Reply by tempus-krupa-patel** [reply]
In my case, I have MySQL and Postgres SQL
